src/query-builder/function-module.ts

```diff
@@ -250,11 +285,12 @@ export class FunctionModule<DB, TB extends keyof DB> {
    * Calls the `sum` function for the column given as the argument.
    *
    * If this is used in a `select` statement the type of the selected expression
-   * will be `number | string | bigint` by default. This is because Kysely can't
-   * know the type the db driver outputs. Sometimes the output can be larger than
-   * the largest javascript number and a string is returned instead. Most drivers
-   * allow you to configure the output type of large numbers and Kysely can't
-   * know if you've done so.
+   * will be `number | string | bigint | null` by default. This is because Kysely
```

Comment:
Here too, null is not in the default.

Reply:
Good catch! forgot about aligning the docs.. 😢